Azure/azure-cli
# [Network] `az network application-gateway waf-policy managed-rule exclusion`: Add new subgroup `rule-set` to support per rule exclusions

> Closed https://github.com/Azure/azure-cli/issues/19734

**Description**

Add new subgroup `rule-set` to support per rule exclusions in Application Gateway WAF.

**Testing Guide**

```
$ az network application-gateway waf-policy managed-rule exclusion rule-set add -g {rg} --policy-name {waf} --match-variable RequestHeaderNames --selector-match-operator StartsWith --selector Bing --type OWASP --version 3.2 --group-name {rule_group} --rule-ids 921140 921150
```

---

This checklist is used to make sure that common guidelines for a pull request are followed.
